webapp/pom.xml

```diff
@@ -104,6 +104,17 @@
         </dependency>
 
         <dependency>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-service</artifactId>
```

Comment:
Why do you need this? https://github.com/Altiscale/hive/blob/branch-1.2.0-alti/jdbc/pom.xml#L45